internal/summary/summary.go

```diff
@@ -56,10 +58,25 @@ func printTaskSummary(l *logger.Logger, t *taskfile.Task) {
 }
 
 func printTaskName(l *logger.Logger, t *taskfile.Task) {
-	l.Outf(logger.Default, "task: %s", t.Name())
+	var aliases string
+	if len(t.Aliases) > 0 {
+		aliases = fmt.Sprintf(" (%s)", strings.Join(t.Aliases, " | "))
+	}
+	l.Outf(logger.Default, "task: %s%s", t.Name(), aliases)
 	l.Outf(logger.Default, "")
 }
 
+func printTaskAliases(l *logger.Logger, t *taskfile.Task) {
+	if len(t.Aliases) == 0 {
+		return
+	}
+	l.Outf(logger.Default, "")
+	l.Outf(logger.Default, "aliases:")
+	for _, alias := range t.Aliases {
+		l.Outf(logger.Default, " - %s", alias)
+	}
```

Comment:
I liked the format choice this this 👍

Reply:
I've updated the PR and removed the alternative syntax in favour of this. I've also set the color of aliases to Cyan since the colored output is now merged. Wdyt?